src/m365/teams/commands/chat/chat-message-send.ts

```diff
@@ -181,7 +181,7 @@ class TeamsChatMessageSendCommand extends GraphCommand {
     }
     catch (err) {
       if ((err as Error).message?.indexOf('404') > -1 && retried < 4) {
-        return await this.createConversation(memberEmails, retried + 1);
+        return this.createConversation(memberEmails, retried + 1);
```

Comment:
I'm not sure how this will act when poo hits the fan. It's a nested function that calls itself, we may want to keep the await in here to make sure it awaits each call separately.